static/app/components/replays/table/replayTable.tsx

```diff
@@ -41,6 +45,10 @@ export default function ReplayTable({
 }: Props) {
   const gridTemplateColumns = columns.map(col => col.width ?? 'max-content').join(' ');
   const hasInteractiveColumn = columns.some(col => col.interactive);
+  const location = useLocation();
+  if (!eventView) {
+    eventView = EventView.fromLocation(location);
+  }
```

Comment:
Why do we have multiple places where we generate the eventView? Couldn't we just do it here in ReplayTable? or closer to where we call `generateQueryStringObjectWithPlaylist` -- or even a custom hook that's like `usePlaylistQueryString`